lib/common/client/middleware/request/soap_headers.rb

```diff
@@ -8,6 +8,7 @@ class SOAPHeaders < Faraday::Middleware
           def call(env)
             env.request_headers['Date'] = Time.now.utc.strftime('%a, %d %b %Y %H:%M:%S GMT')
             env.request_headers['Content-Length'] = env.body.bytesize.to_s
+            Rails.logger "[SOAPHeaders] Environment: #{env}" if Settings.test_user_dashboard.env == 'dev'
```

Comment:
I think you may be able to use the [Settings.vsp_environment yaml setting](https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/deployment/config/vets-api/dev-settings.local.yml.j2#L2) here, but `== 'dev'` will need to be `== 'development'` It doesn't appear that the test_user_dashboard setting is defined in [the dev settings in the devops repo](https://github.com/department-of-veterans-affairs/devops/blob/master/ansible/deployment/config/vets-api/dev-settings.local.yml.j2)

Reply:
Just confirmed, TUD is only available in staging and only has test_user_dashboard.env set in the [staging yml](https://github.com/department-of-veterans-affairs/devops/blob/f7d2dd14c05d85c6538c748c4953a9517267c6c3/ansible/deployment/config/vets-api/staging-settings.local.yml.j2#L519)

Reply:
ah yes, you're right. I should have checked. I made this update